client/cli/cli.go

```diff
@@ -87,6 +87,31 @@ func Run(args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
+
+	// set command completer
+	t.line.SetWordCompleter(func(line string, pos int) (head string, completions []string, tail string) {
+		start := strings.LastIndex(line[:pos], " ")
+		match := ""
+		if start < 0 { // this is the command to match
+			head = ""
+			tail = line[pos:]
+			match = line
+		} else if strings.HasPrefix(line, "help ") {
+			head = line[:start+1]
+			match = line[start+1:]
+			tail = line[pos:]
+		} else {
```

Comment:
Could this just be a guard clause that if `pos == 0` or `line == ""` instead of an else that returns.